src/server.ts

```diff
@@ -0,0 +1,93 @@
+import { Client } from "pg";
+import { config } from "dotenv";
+import express from "express";
+import cors from "cors";
+import filePath from "./filePath";
+
+config(); //Read .env file lines as though they were env vars.
+
+//Call this script with the environment variable LOCAL set if you want to connect to a local db (i.e. without SSL)
+//Do not set the environment variable LOCAL if you want to connect to a heroku DB.
+
+//For the ssl property of the DB connection config, use a value of...
+// false - when connecting to a local DB
+// { rejectUnauthorized: false } - when connecting to a heroku DB
+const herokuSSLSetting = { rejectUnauthorized: false };
+const sslSetting = process.env.LOCAL ? false : herokuSSLSetting;
+const dbConfig = {
+  connectionString: process.env.DATABASE_URL,
+  ssl: sslSetting,
+};
+
+const app = express();
+
+app.use(express.json()); //add body parser to each following route handler
+app.use(cors()); //add CORS support to each following route handler
+
+const client = new Client(dbConfig);
+client.connect();
+
+app.get("/", async (req, res) => {
+  const pathToFile = filePath("../public/index.html");
+  res.sendFile(pathToFile);
+});
+
+app.get("/breeds", async (req, res) => {
+  try {
+    const queryRes = await client.query(
+      "SELECT * FROM breedvotes ORDER BY vote DESC, dogbreed"
+    );
+    res.status(200).json(queryRes.rows);
+  } catch (error) {
+    res.status(404).send(error.stack);
+  }
+});
+
+app.post<{}, {}, { dogbreed: string }>("/breeds", async (req, res) => {
+  try {
+    const newBreed = req.body.dogbreed;
+    const query = `INSERT INTO breedvotes (dogbreed)  
+                VALUES ($1)
+                Returning *`;
+    const queryRes = await client.query(query, [newBreed]);
+    if (queryRes.rowCount === 0) {
+      res.status(400);
```

Comment:
Good separating the errors to give more information. Send a response body back to avoid client hanging